sycl/source/detail/kernel_impl.cpp

```diff
@@ -46,7 +152,7 @@ bool kernel_impl::isCreatedFromSource() const {
   // kernel SecondKernel = kernel(ClKernel, Context);
   // clReleaseKernel(ClKernel);
   // FirstKernel.isCreatedFromSource() != FirstKernel.isCreatedFromSource();
-  return IsCreatedFromSource;
+  return MIsCreatedFromSource;
```

Comment:
```suggestion
  return MCreatedFromSource;
```
To look more like a statement